glustercli/cmd/snapshot-info.go

```diff
@@ -49,6 +49,7 @@ func snapshotInfoDisplay(snap api.SnapGetResp) {
 	fmt.Println("Origin Volume name:", snap.ParentVolName)
 	fmt.Println("Snap Creation Time:", snap.SnapTime)
 	fmt.Println("Labels:", "To Be Added")
+	fmt.Println("Snapshot Description:", snap.Description)
```

Comment:
I just happened to see commented out code. That needs to be fixed or removed.

@rafikc30 Can you take care of it ? Git blame pointed me to https://github.com/gluster/glusterd2/pull/533

Reply:
Sure. I will do that